Description: revert persistence
 .
 liveusbcreator (0.1-7) unstable; urgency=medium
 .
   * fix Debian control and dep
Author: arnault perret <arpinux@member.fsf.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- liveusbcreator-0.1.orig/liveUSBcreator
+++ liveusbcreator-0.1/liveUSBcreator
@@ -1,15 +1,14 @@
-#!/bin/sh
+#!/bin/bash
 #Auteur :      coyotus, thuban 
 #licence :     GNU General Public Licence v3
 
 # Description : création simplifiée de live usb
 # Usage : usbmaker </path/to/iso> (facultative)
-# Depends : zenity, coreutils, pv , util-linux
+# Depends : zenity, coreutils, pv
 
 # 
 ISO=""
 USB=""
-DOPERSIST=""
 
 # messages
 CHOICE="Choice"
@@ -26,8 +25,6 @@ CHOICE="Choice"
 MODEL="Model"
 SIZEMSG="Size"
 CAREFUL="WARNING!\nThis tool will erase the entire selected disk.\n Don't forget to make a backup."
-NORMALLIVE="Create a normal live usb"
-PERSISTENT="Create a live usb with data persistence"
 
 case $LANG in
 	fr*)
@@ -43,8 +40,6 @@ case $LANG in
         MODEL="Modèle"
         SIZEMSG="Taille"
         CAREFUL="ATTENTION!\nCet outil va intégralement effacer votre clé USB.\n N'oubliez pas de sauvegarder vos données."
-        NORMALLIVE="Créer une clé USB live classique"
-        PERSISTENT="Créer une clé USB avec persistance des données"
         ;;
     nl*)
         APPNAME="Handy live usb creator"
@@ -67,28 +62,9 @@ copytousb() {
     # this is run as root
     # $1 is iso path
     # $2 is device path i.e. /dev/sda
-    # $3 : if = "Y", a new partition for persistence is created
     ( pv -n "$1" |dd of="$2" bs=4M oflag=sync ) 2>&1 | zenity --title="$APPNAME" --text="$COPYMSG" --width 400 --progress --auto-close
     RET=$?
     sync # just to be sure
-
-    if [ "$3" = "Y" ]; then
-        echo "prepare persistence"
-        echo "n
-p
-2
-
-
-w
-" | fdisk $2
-        mkfs.ext4 -F -L "persistence" "${2}2"
-        mkdir -p /tmp/liveusbcreator
-        mount -t ext4 "${2}2" /tmp/liveusbcreator
-        echo "/usr union" >> /tmp/liveusbcreator/persistence.conf
-        echo "/home" >> /tmp/liveusbcreator/persistence.conf
-        umount /tmp/liveusbcreator
-    fi
-
     if [ $RET -eq 0 ]; then
         zenity --info --title="$APPNAME" --text="$SUCCESS"
     else
@@ -97,20 +73,9 @@ w
 }
 
 if [ "$(whoami)" != "root" ]; then
-    rep=$(zenity --width=375 --list --title "LiveUSBcreator" --text "$CAREFUL"  --radiolist \
-        --column="$CHOICE" --column="" \
-        "choix" "$NORMALLIVE" \
-        "choix" "$PERSISTENT")
-
-    if [ $? -ne 0 ]; then
-        exit 0
-    fi
-    if [ "$rep" = "$PERSISTENT" ]; then
-        DOPERSIST="Y"
-    fi
+    zenity --info --title="LiveUSBcreator" --text="$CAREFUL"
 fi
 
-# où est l'image iso?
 if [ -z "$1" ]; then
     ISO="$(zenity --title="$ISOMSG" --file-selection --file-filter="Disc Image *.iso | *.iso")"
     if [ $? -ne 0 ]; then
@@ -120,7 +85,6 @@ else
     ISO="$1"
 fi
 
-# où est la clé usb?
 if [ -z "$2" ]; then
     USBKEYS=$(
     grep -Hv ^0$ /sys/block/*/removable |
@@ -147,13 +111,6 @@ else
     USB="$2"
 fi
 
-# on crée un live persistent
-if [ -n "$3" ]; then
-    DOPERSIST="$3"
-fi
-
-
-# gestion d'erreurs
 if [ -z "$ISO" ] || [ -z "$USB" ]; then
     echo "arguments incohérents"
     exit 1
@@ -163,14 +120,12 @@ elif [ ! -b "$USB" ]; then
 elif [ ! -f "$ISO" ]; then
     echo "l'image iso donnée n'existe pas"
     exit 1
-
-# partie où l'iso est copiée
-elif [ "$(whoami)" = "root" ]; then
-    copytousb "$ISO" "$USB" "$DOPERSIST"
+elif [ "$(whoami)" == "root" ]; then
+    copytousb "$ISO" "$USB"
     exit 0
 fi
 
 
-gksudo $0 "$ISO" "$USB" "$DOPERSIST"
+gksudo $0 "$ISO" "$USB"
 
 exit 0
